.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,43 @@
+name: Feature request
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Suggest an idea for this project
+
+title: "[ENH] "
+
+labels: ["enhancement"]
```

Comment:
we don't yet have a good label for this. Maybe `"feedback:feature request"` would fit well.

Reply:
done